src/site/xdoc/server/quick-start-cassandra.xml

```diff
@@ -56,18 +54,22 @@ Step 3: Compile
   
 <i>$ mvn package -DskipTests -am -pl server/container/guice/cassandra-guice</i>
 
+  * Compile the James CLI project
+  
+<i>$ mvn package -DskipTests -am -pl server/container/cli</i>
+
 Step 3: Deploy
 ##############
 
 3.1. Deploy Cassandra (optional)
 You may skip this part if you already have a running Cassandra on your network.
 
-<i>$ docker run --name=cassandra cassandra:2.2.3</i>
+<i>$ docker run -d --name=cassandra cassandra:2.2.3</i>
 
 3.2. Deploy ElasticSearch (optional)
 You may skip this part if you already have a running ElasticSearch on your network.
 
-<i>$ docker run --name=elasticsearch elasticsearch:2.2.1</i>
+<i>$ docker run -d --name=elasticsearch elasticsearch:2.2.1</i>
 
 Step 4: Configure
 #################
```

Comment:
in this section, I suggest to document how to start from sample files, it's what we advise people to do anyway